src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,208 @@
+= Why Hawkular?
+Heiko Rupp
+2015-07-04
+:description: Vision for Hawkular
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
```

Comment:
My take, based on yours: "To be the most effective solution to monitor and manage applications deployed on Wildfly and derivatives, as well as other Java-based Middleware tools."